volttrontesting/fixtures/docker_wrapper.py

```diff
@@ -65,11 +65,11 @@ def create_container(image_name: str, ports: dict = None, env: dict = None, comm
                 break
             time.sleep(0.1)
             container.reload()
-
-        if invalid:
-            yield None
-        else:
-            yield container
-
-        container.kill()
+        try:
```

Comment:
I know that I have another PR (#2388) that adds this try/except, but I need it here so that the test can automatically cleanup containers.